Tests/test_main.py

```diff
@@ -0,0 +1,21 @@
+"""Test the main module"""
+import unittest
+from life_expectancy.main import Country
+
+def test_list_country():
+    """Test Country list function"""
+
+    case = unittest.TestCase()
+    countries_list_actual = Country.list()
+    countries_list_expected = [
```

Comment:
**Question**: Why are you using a TestCase here? Wouldn't this be simpler:

```python
def test_list_country():
    """Test Country list function"""
    expected = [
        "AL",
        "AM",
        # rest of list...
        "XK",
    ]
    assert Country.list() == expected
```

Reply:
But using the equal takes into consideration the order of the list and the test would fail if, for some region countries were in a different order which in my Enum is indifferent the order they are on. Additionally, if I were to change the list function to print is some specific order this test would then fail.

Reply:
There's an easier way to make it clear that the list doesn't matter, using sets:

```suggestion
    difference = set( Country.list()) ^ set(expected )
    assert not difference
```
It's strange to see a pytest test suite using unittest.TestCase